tex/chapters/Introduction.tex

```diff
@@ -1,50 +1,163 @@
-        \includegraphics[width=.6\textwidth]{../img/plot1.pdf}
-        \caption{The great data}\label{fig:the_data}
-    \end{center}
-\end{figure}
+\begin{equation}\label{eq:payoff_matrix_symb}
+% 
+P = \bordermatrix{~ & C & D \cr
+                  C & (R, R) & (T, S) \cr
+                  D & (S, T) & (P, P) \cr}
+% 
+\end{equation}
 
-\section{The distribution of the data}\label{sec:the_distribution_of_the_data}
+There are also two assumptions that need to be stated.
+Firstly, both players are rational.
+Secondly, there is no communication between them.
+It is then easy to see that regardless of the choice of one player, the other will always obtain a higher payoff by defecting instead of cooperating.
+Therefore we have a pure Nash Equilibrium where both players defect, despite the fact that both players would do better if they were to cooperate with each other.
+Additionaly, to ensure this behaviour occurs, the payoffs must satisfy the following inequalities:
 
-Figure shows the distribution of the data.
+\begin{equation}\label{eq:intuitive}
+S < D < C < T
+\end{equation}
 
-\begin{figure}[!hbtp]
-    \begin{center}
-        \includegraphics[width=.6\textwidth]{../img/plot2.pdf}
-        \caption{The distribution of the great data}\label{fig:the_distribution_of_the_data}
-    \end{center}
-\end{figure}
+and
 
-\lipsum
+\begin{equation}\label{eq:ensure_coop}
+(S + T) < 2 C
+\end{equation}
 
-%!TEX root = ../main.tex
+Equation \ref{eq:intuitive} merely fixes the payoffs in their intuitive order.
+Equation \ref{eq:ensure_coop} ensures that alternating between cooperating and defecting (players take it in turns to stab each other in the back) performs no better than mutual cooperation.
+These inequalities allow for many different payoff matrices to be formulated, but values of $(R, S, T, P) = (3, 0, 5, 1)$ are commonly used in literature (find references).
+We can now formulate the new payoff matrix, as shown in Figure \ref{eq:payoff_matrix}
+A more detailed explanation of the Prisoner's Dilemma is given in ~\cite{Gotts2003}.
+
+\begin{equation}\label{eq:payoff_matrix}
+% 
+P = \bordermatrix{~ & C & D \cr
+                  C & (3, 3) & (5, 0) \cr
+                  D & (0, 5) & (1, 1) \cr}
+% 
+\end{equation}
 
-\section{Conclusion}\label{sec:conclusion}
+\section{Problem Description}
+
+As previously mentioned, the Axelrod-Python library contains 139 strategies, significantly more than the 13 and 64 strategies submitted to Axelrod's first and second tournaments.
+This now raises the issue of duplication, and subsequently, how to tell the difference between strategies.
+Currently the most popular method is Fingerprinting which produces a visual representation of a strategy, an example is shown in Figure.
+
+The issue with this method is that it relies on knowledge of the underlying markov chain of the strategy which cannot be easily constructed from the source code.
+
+More needed once we know what's actually gonna happen.
```

Comment:
Yeah this section will lead a little more. I think perhaps, here you just have an image of a fingerprint. Then potentially have another chapter that goes over the paper in brutal detail.